test/integration/production/test/index.test.js

```diff
@@ -133,6 +133,7 @@ describe('Production Usage', () => {
             /node_modules\/react\/cjs\/react\.production\.min\.js/,
             /next\/router\.js/,
             /next\/dist\/client\/router\.js/,
+            /next\/dist\/shared\/lib\/router\/utils\/resolve-rewrites\.js/,
```

Comment:
Should this be added to a `notTests` for another page?

Reply:
No, it should be included in all of the tested pages since they use `next/router` or `next/link`, I updated tests to reflect this in https://github.com/vercel/next.js/pull/29673/commits/6aeccbf3900019e55d8f221a6390434947b99840

Reply:
Perhaps we need a new page that doesn't use `next/router` or `next/link`